sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TextAnalyticsClientTestBase.java

```diff
@@ -0,0 +1,428 @@
+                    .filter(element -> element instanceof DetectLanguageResult)
+                    .map(element -> (DetectLanguageResult) element)
+                    .collect(Collectors.toList());
+
+                final List<DetectLanguageResult> actualResultList = actualResult.stream()
+                    .filter(element -> element instanceof DetectLanguageResult)
+                    .map(element -> (DetectLanguageResult) element)
+                    .collect(Collectors.toList());
+                assertEquals(expectedResultList.size(), actualResultList.size());
+
+                actualResultList.forEach(actualItem -> {
+                    Optional<DetectLanguageResult> optionalExpectedItem = expectedResultList.stream().filter(
+                        expectedEachItem -> actualItem.getId().equals(expectedEachItem.getId())).findFirst();
+                    assertTrue(optionalExpectedItem.isPresent());
+                    DetectLanguageResult expectedItem = optionalExpectedItem.get();
+                    if (actualItem.getError() == null && this.showStatistics) {
+                        validatePrimaryLanguage(expectedItem.getPrimaryLanguage(), actualItem.getPrimaryLanguage());
+                        validateDocumentStatistics(expectedItem.getStatistics(), actualItem.getStatistics());
+                        validateDetectedLanguages(expectedItem.getDetectedLanguages(), actualItem.getDetectedLanguages());
+                    }
+                });
+                break;
+            default:
+                break;
+        }
+    }
+
+    /**
+     * Helper method to verify TextBatchStatistics.
+     *
+     * @param expectedStatistics
+     * @param actualStatistics
+     */
+    private static void validateBatchStatistics(TextBatchStatistics expectedStatistics,
+                                                TextBatchStatistics actualStatistics) {
+        assertEquals(expectedStatistics.getDocumentCount(), actualStatistics.getDocumentCount());
+        assertEquals(expectedStatistics.getErroneousDocumentCount(), actualStatistics.getErroneousDocumentCount());
+        assertEquals(expectedStatistics.getValidDocumentCount(), actualStatistics.getValidDocumentCount());
+        assertEquals(expectedStatistics.getTransactionCount(), actualStatistics.getTransactionCount());
+    }
+
+    /**
+     * Helper method to verify the error document.
+     *
+     * @param expectedError the Error returned from the service.
+     * @param actualError the Error returned from the API.
+     */
+    static void validateErrorDocument(Error expectedError, Error actualError) {
+        assertEquals(expectedError.getCode(), actualError.getCode());
+        assertEquals(expectedError.getMessage(), actualError.getMessage());
+        assertEquals(expectedError.getTarget(), actualError.getTarget());
+        assertEquals(expectedError.getInnererror(), actualError.getInnererror());
```

Comment:
getInnerError? This may be an autogenerated class, but the casing is wrong.

Reply:
Error model would be revisited in this https://github.com/azure/azure-sdk-for-java/issues/6559
Can handle this in that one.

Reply:
Can you update the issue to at least reference this comment or update the description? I can see this being lost in all the updates.

Reply:
updated.